shell/platform/windows/keyboard_key_handler.cc

```diff
@@ -95,7 +96,7 @@ void KeyboardKeyHandler::ResolvePendingEvent(uint64_t sequence_id,
     }
   }
   // The pending event should always be found.
-  assert(false);
+  FML_LOG(FATAL) << "No unreplied pending events found";
```

Comment:
Consider something like:

```suggestion
  FML_LOG(FATAL) << "Could not find pending key event " << sequence_id;
```

Reply:
Oh nice - much better! Done!